peerpod-ctrl/controllers/peerpod_controller.go

```diff
@@ -83,3 +107,44 @@ func (r *PeerPodReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&confidentialcontainersorgv1alpha1.PeerPod{}).
 		Complete(r)
 }
+
+func (r *PeerPodReconciler) runtimeCloudConfig() error {
+	peerpodscm := corev1.ConfigMap{}
+	peerpodssecret := corev1.Secret{}
+	ns := os.Getenv("PEERPODS_NAMESPACE")
+	if ns == "" {
+		return fmt.Errorf("PEERPODS_NAMESPACE is not set")
+	}
+
+	if err := r.Client.Get(context.TODO(), types.NamespacedName{Name: ppConfigMap, Namespace: ns}, &peerpodscm); err == nil {
+		for k, v := range peerpodscm.Data {
+			os.Setenv(k, v)
```

Comment:
Do you need to set all the k,v pairs or just need the CLOUD_PROVIDER name ?

Reply:
I need to set all of those required for cloud provider authentication, later, we may be able to change the way we are getting the the configuration in the cloud pkg and also use it here instead of using env vars

Reply:
Ok, can you please add the above as a comment in the code.